ocpi-toolkit-2.2.1/src/main/kotlin/com/izivia/ocpi/toolkit/common/OcpiExceptions.kt

```diff
@@ -14,6 +14,11 @@ class OcpiResponseException(
 ) : Exception("Ocpi error: ${statusCode.toOcpiStatus()} $statusCode ($statusMessage)")
 
 // 2xxx: Client errors
+class OcpiObjectNotFoundException(
```

Comment:
I'm not entirely sure, why we have OcpiException as sealed class. It doesn't look like we are trying to catch the individual sub types anywhwere, so a single OcpiException class, combined with a few helper methods creating the exceptions seems to be sufficient. It kind of forces us to have two classes, the sealed one, and the (untyped) response exception. Even though both transport the same information.